demo/index.html

```diff
@@ -20,7 +20,41 @@
       <h3>Basic mtz-file-download-behavior demo</h3>
       <demo-snippet>
         <template>
-          <mtz-file-download-behavior></mtz-file-download-behavior>
+
+          <file-downloader></file-downloader>
+
+          <dom-module id="file-downloader">
+            <template>
+              <button on-click="openUriInNewTab">Open In New Tab</button>
+              <button on-click="openUriInCurrentTab">Open In Current Tab</button>
+              <button on-click="createFile">Create File</button>
+            </template>
+            <script>
+              window.addEventListener('WebComponentsReady', () => {
+                Polymer({
+                  is: 'file-downloader',
+                  behaviors: [
+                    // Register use of the behavior
+                    mtz.FileDownloadBehavior
+                  ],
+                  // Open data at a URI in a new tab or the current tab allowing the user
+                  // to save if they so desire.
+                  openUriInNewTab() {
+                    this.downloadFromURI('./sample.txt', true);
+                  },
+                  openUriInCurrentTab() {
+                    this.downloadFromURI('./sample.txt', false);
+                  },
+                  // Serialize a string of data to a <mime-type> file.
+                  createFile() {
+                    const data = "You've serialized data to file. Happy days!!!";
```

Comment:
I would use `iron-ajax` to fetch a `csv` from this repository and use the same `csv` in the `downloadFromURI`